buildkitd/buildkitd.go

```diff
@@ -125,28 +135,28 @@ func ResetCache(ctx context.Context, console conslogging.ConsoleLogger, image, c
 
 // MaybeStart ensures that the buildkitd daemon is started. It returns the URL
 // that can be used to connect to it.
-func MaybeStart(ctx context.Context, console conslogging.ConsoleLogger, image, containerName string, settings Settings, opts ...client.ClientOpt) (string, error) {
-	isStarted, err := IsStarted(ctx, containerName)
+func MaybeStart(ctx context.Context, console conslogging.ConsoleLogger, image, containerName string, fe containerutil.ContainerFrontend, settings Settings, opts ...client.ClientOpt) (string, error) {
+	isStarted, err := IsStarted(ctx, containerName, fe)
 	if err != nil {
 		return "", errors.Wrap(err, "check is started buildkitd")
 	}
 	if isStarted {
 		console.
 			WithPrefix("buildkitd").
 			Printf("Found buildkit daemon as docker container (%s)\n", containerName)
```

Comment:
Could we parameterize this `Printf` to say `Found buildkit daemon as podman container` etc depending on frontend? Alternatively, we could also report the frontend completely separately, at the very start of Earthly.